gpii/node_modules/flowManager/src/FlowManager.js

```diff
@@ -41,6 +41,7 @@ require("gpii-user-errors");
 fluid.defaults("gpii.flowManager", {
     gradeNames: ["kettle.app"],
     components: {
+        // TODO: Make this use the solutions registry data source by default?
```

Comment:
But this isn't actually a dataSource, just a placeholder definition, right?

Reply:
The hard-coded encoding in the placeholder clobbered the encoding from the mixed in type, which is why I noticed.  We never use another type, why can't we just trim the config and update this to use the mixed in type?